docs: ot2 installation and relocation

jwwojak opened the pull request:
# Overview

This migrates the OT-2 setup instructions into MkDocs. 

Accidentally includes box contents section (`box-contents.md`). Didn't mean to, but here we are.

Sandbox should be: https://sandbox.docs.opentrons.com/docs-ot2-install-relocate/

## Test Plan and Hands on Testing

Please review for clarity and accuracy. 

## Changelog

This adds 3 new files (sorry, things get going and I forget to keep files separate)

- requirements.md: the things you need to have/do before setting up OT-2
- unboxing.md: instructions and images in an ordered list
- box-contents.md: to handle some other text that's been moved out of the installation section.

Original images are untouched. Many have large padded margins, which puts almost too much whitespace around them. However, this is good enough for an old manual. Not sure it's worth it to remove this padding. They are clear and visible.

## Review requests

Text, grammar, code?

## Risk assessment

Medium.

jwwojak committed:
Preliminary adjustments

jwwojak committed:
Prerequisites start

jwwojak committed:
Remove this

jwwojak committed:
Reorganizing files and .yml nav

jwwojak committed:
Workspace and power requirements

jwwojak committed:
Environmental conditions

- temp
- humidity
- altitude

jwwojak committed:
pre-meeting save

jwwojak committed:
Merge branch 'docs-ot2-manual-revisions' into docs-ot2-install-relocate

jwwojak committed:
Afternoon commit

- adds intro, contents, and some assembly steps
- using OT-2 quickstart

jwwojak committed:
More installation and assembly steps

jwwojak committed:
Lunchtime save - image uploads and instructions

jwwojak committed:
End of day - procedure images

jwwojak committed:
Images - finding and sorting things out

jwwojak committed:
Found a missing image

Shows tape around the calibration switch cover.

jwwojak committed:
Robot and deck map images with labels

Looks better on a large monitor.

jwwojak committed:
box content images

jwwojak committed:
labeled images and features

These were a major PITA.

jwwojak committed:
Remove dup headers and resize

Product elements was duplicated. Remove and make H3s into H2s.

jwwojak committed:
A few adjustments before review

ecormany commented on docs/ot-2/mkdocs.yml:
```diff
@@ -4,6 +4,9 @@ nav:
   - Safety and Regulatory Information: regulatory.md
+  - Product Elements: box-contents.md
+  - Installation Requirements: ot2-installation/requirements.md
+  - Unboxing and Assembly: ot2-installation/unboxing.md
```
Probably mirror the structure for Flex, with a landing page and subpages. Flex YML file (which has more subpages): 
```
  - Installation and Relocation: 
      - Installation and Relocation: installation/index.md
      - Installation Requirements: installation/requirements.md
      - Unboxing: installation/unboxing.md
      - First Run: installation/first-run.md
      - Instrument Installation and Calibration: installation/instruments.md
      - Relocation: installation/relocation.md
```

And Flex side nav output:
<img width="237" height="180" alt="Image" src="https://github.com/user-attachments/assets/402f88e9-da9f-4ffe-9b93-e1d5f5ce2c09" />

jwwojak replied:
I had this and got rid of it for some reason. Will restore.

ecormany commented on docs/ot-2/docs/installation/requirements.md:
```diff
@@ -0,0 +1,41 @@
+
+- **Weight bearing:** The OT-2 weighs 42 kg (93 lbs). Place the robot on a surface that can support its weight plus the weight of any modules, labware, liquids, or other lab equipment that may be used in your work.
+
+- **Bench surface:** A good workspace is stationary, sturdy, level, and liquid-resistant. Tables or benches with wheels (even locking wheels) are not recommended. OT-2 gantry movements can shake light weight or moveable tables.
```
This language was written for Flex, so it's conservative (but not bad) to apply it to OT-2.

ecormany commented on docs/ot-2/docs/ot2-installation/requirements.md:
```diff
@@ -0,0 +1,41 @@
+
+- **Bench surface:** A good workspace is stationary, sturdy, level, and liquid-resistant. Tables or benches with wheels (even locking wheels) are not recommended. OT-2 gantry movements can shake light weight or moveable tables.
+
+- **Operating space:** The OT-2 needs a working area with minimum height, depth, and width of 66 cm x 57 cm x 63 cm (≈26" x 23" x 25").
```
I would clarify that these are the exact dimensions of the robot. Readers may home in on the numbers and miss the suggestion below to leave extra clearance.

jwwojak replied:
Will add text to indicate extra space is required.

> The OT-2's dimensions are 66 cm x 57 cm x 63 cm (≈ 26" x 22.5" x25"). The working area must include extra clearance on the sides and back beyond the space occupied by the robot. This additional space is needed to accommodate cables and USB connections, and to allow for proper exhaust dissipation from modules that heat or cool.

Followed by an image that shows dimensions.

ecormany commented on docs/ot-2/docs/ot2-installation/requirements.md:
```diff
@@ -0,0 +1,41 @@
+The total power consumption of the OT-2 varies depending on the requirements of your protocol.
+
+- **Operating:** ~ 100–180 W
+- **Idle:** ~ 90–120 W
```
Format nitpick
```suggestion
- **Operating:** ~100–180 W
- **Idle:** ~90–120 W
```

jwwojak replied:
Will fix.

ecormany commented on docs/ot-2/docs/ot2-installation/requirements.md:
```diff
@@ -0,0 +1,41 @@
+- The robot is designed for indoor use only.
+- Temperature range: 20 °C ± 4° C (68 °F ± 7.2 °F).
+- Relative humidity: 40-60% (up to 80% is acceptable, but outside the recommended range).
+- Altitude: Up to 2,000 meters above sea level.
```
I would either 
- bold these bullet headers, like in the above sections
- convert this to a table like we have for Flex and module docs

jwwojak replied:
Will add **bold** bullet headers. An unordered list matches that manual.

ecormany commented on docs/ot-2/docs/ot2-installation/unboxing.md:
```diff
@@ -0,0 +1,90 @@
+
+Congratulations! Your OT-2 has arrived and you've prepared a space for it in your lab. The information in this section provides instructions that walk you through the steps required to get the OT-2 unboxed, assembled, and ready for use.
+
+## Effort and time required
```
This doesn't really merit an H2 for a couple sentences. You can just delete the header line and let the page intro have two paragraphs.

jwwojak replied:
Applied suggestion as fix.

ecormany commented on docs/ot-2/docs/ot2-installation/unboxing.md:
```diff
@@ -0,0 +1,90 @@
+
+<div class="instruction-list" markdown>
+
+<!-- Remove padding/whitespace around images or good enough? -->
```
Images are fine for now, imo.

ecormany commented on docs/ot-2/docs/ot2-installation/unboxing.md:
```diff
@@ -0,0 +1,90 @@
+
+14. Perform the following assembly check:
+    - Confirm that 3 transport fasteners have been removed, as powering on the robot with these brackets attached can damage the robot.
+    - Confirm that you are not missing any required cables or parts. If you are missing components, contact Customer Support.
```
"Opentrons Support" per house style, here and below. And probably a link to do so on the first mention.

jwwojak replied:
Change all to "Opentrons Support." Link on first mention to https://opentrons.com/opentrons-support.

ecormany commented on docs/ot-2/docs/ot2-installation/unboxing.md:
```diff
@@ -0,0 +1,90 @@
+15. Clean and sanitize the robot. <font color="red">REMINDER: LINK TO CLEANING SECTION HERE</FONT>
+
+    !!!note
+        When during cleaning or other maintenance, be sure to turn off the machine and disconnect the power before proceeding. Also, pay attention to any warning labels as you work.
```
Looks like a rephrasing got caught halfway between here. Go with either "When cleaning or performing other maintenance…" or "During cleaning or other maintenance…"

Also, "machine" is weird copy. We call it a "robot".

ecormany commented on docs/ot-2/docs/ot2-installation/unboxing.md:
```diff
@@ -0,0 +1,90 @@
+17. Connect the Ethernet cable to the OT-2 and laptop. If your computer does not have an Ethernet port, use the provided Ethernet-to-USB dongle.
+    ![laptop USB-Ethernet connection](../images/computer-connection.png)
+
+18. Connect the external power supply to the OT-2 and then connect the main power cable to a wall outlet.
```
"mains" as in "mains power"? or "main" as in "primary"?

jwwojak replied:
I'm striking "main". Will use "connect the power cable to a wall outlet."

ecormany commented on docs/ot-2/docs/installation/unboxing.md:
```diff
@@ -0,0 +1,90 @@
+
+</div>
+
+At this point your OT-2 is ready for use. You can attach pipettes to the gantry, add modules to the deck, and create protocols in for the robot using the Opentrons App.
```
Another halfway between phrasing. You can't create protocols in the app (yet) so maybe
```suggestion
At this point your OT-2 is ready for use. You can attach pipettes to the gantry, add modules to the deck, and upload protocols to the robot using the Opentrons App.
```

ecormany commented on docs/ot-2/docs/images/ot2-deck-map.png:
"Adapters allow existing labware to fit on deck" is curious. It makes it sound like you _can't_ place labware straight on deck. If it's not too difficult, maybe we can remove it.

jwwojak replied:
Removing that text and pointer.

ecormany commented on docs/ot-2/docs/box-contents.md:
```diff
@@ -0,0 +1,105 @@
+</figure>
+
+<figure markdown>
+![T10 Torx](images/torx-t10.png "T10 Torx")
```
```suggestion
![T10 Torx](images/torx-t10.png "T10 Torx Key")
```

ecormany commented on docs/ot-2/docs/box-contents.md:
```diff
@@ -0,0 +1,105 @@
+
+<figure markdown>
+![Hex L-key set](images/hex-key-set.png "Multiple size hex L-keys")
+<figcaption>(4) Hex Keys (1.5 mm, 2 mm, 2.5 mm, 3 mm)</figcaption>
```
I made a CSS rule that you can apply here for nicer formatting:

```suggestion
<figcaption>(4) Hex Keys
<p class="part-info">(1.5 mm, 2 mm, 2.5 mm, 3 mm)</p></figcaption>
```

ecormany reviewed (commented):
Nice! Just small suggestions.

jwwojak committed:
Forgot to add a small relocation piece

jwwojak committed:
Revisions from review comments

Some TOC reorog and text, image changes

jwwojak reviewed (commented):
test

jwwojak committed:
Apply suggestion from @ecormany

Co-authored-by: Ed Cormany <edward.cormany@opentrons.com>

jwwojak committed:
Update docs/ot-2/docs/box-contents.md

Co-authored-by: Ed Cormany <edward.cormany@opentrons.com>

jwwojak committed:
Update docs/ot-2/docs/installation/unboxing.md

Co-authored-by: Ed Cormany <edward.cormany@opentrons.com>

jwwojak committed:
Afternoon commit

Adding new sections to match Flex setup instructions. Adding related content.

jwwojak committed:
Power and computer connection and images

jwwojak committed:
Network connections sections lunch save

jwwojak committed:
First run - wifi, LAN, software update, naming

jwwojak committed:
Installation index section

jwwojak committed:
Starting instruments and calibration

jwwojak committed:
Pipette comparison and images

jwwojak committed:
GEN1 pipette images potato quality

jwwojak committed:
Pre standup commit

jwwojak committed:
Commit rather than stash

jwwojak committed:
End of day commit

Pipette installation instructions.

jwwojak committed:
Final changes and commit

Will open PR

jwwojak commented:
The app has nice, detailed instructions and images for pipette installation and calibrations. I made the manual chapter on this short because all this info is in the app. It's basically, "click install pipette and follow instructions in the app." The manual doesn't need to duplicate what's already documented.

jwwojak committed:
Add class rule for ordered list

ecormany commented on docs/ot-2/docs/installation/index.md:
```diff
@@ -0,0 +1,11 @@
+
+- Before taking delivery of your OT-2, make sure your lab or facility meets the all the criteria in the [Installation Requirements](requirements.md) section.
+
+- When it's time to get your OT-2 up and running, follow the instructions in the [Unboxing](unboxing.md), [First Run](first-run.md), and [Instruments and Calibration](instruments.md) sections.
```
```suggestion
- When it's time to get your OT-2 up and running, follow the instructions in the [Unboxing](unboxing.md), [First Run](first-run.md), and [Instrument Installation and Calibration](instruments.md) sections.
```

ecormany commented on docs/ot-2/docs/installation/requirements.md:
```diff
@@ -2,20 +2,18 @@
 
 - **Weight bearing:** The OT-2 weighs 42 kg (93 lbs). Place the robot on a surface that can support its weight plus the weight of any modules, labware, liquids, or other lab equipment that may be used in your work.
 
 - **Bench surface:** A good workspace is stationary, sturdy, level, and liquid-resistant. Tables or benches with wheels (even locking wheels) are not recommended. OT-2 gantry movements can shake light weight or moveable tables.
```
```suggestion
- **Bench surface:** A good workspace is stationary, sturdy, level, and liquid-resistant. Tables or benches with wheels (even locking wheels) are not recommended. OT-2 gantry movements can shake lightweight or moveable tables.
```

ecormany commented on docs/ot-2/docs/installation/unboxing.md:
```diff
@@ -0,0 +1,74 @@
+2. Carefully cut open the blue plastic shipping bag. Be sure to avoid cutting any robot components inside.
+    ![cutting the shipping bag](../images/cut-shipping-bag.png)
+
+3. Pull upwards using the black steel frame to remove the robot from the shipping bag and box. The OT-2 weighs about 42 kg (93 lbs), always get help from a lab partner to remove the robot from the box.
```
splice! either "…(93 lbs). Always…" or "Because the OT-2 weighs…" would fix it.

jwwojak replied:
Ok, will correct this:

> Pull upwards using the black steel frame to remove the robot from the shipping bag and box. Because the OT-2 weighs about 42 kg (93 lbs), always get help from a lab partner to lift it.

ecormany commented on docs/ot-2/docs/installation/unboxing.md:
```diff
@@ -0,0 +1,74 @@
+
+At this point you should have a fully assembled robot, but it isn't time to power it up and start running protocols, at least not yet. Before going forward, check your work against the items in this list:
+
+  - Confirm that all 3 transport fasteners have been removed, as powering on the robot with these brackets attached can damage the robot.
```
would be fun to have a checklist CSS style here. totally optional though.

jwwojak replied:
🤔  hmmmmm will try.

ecormany commented on docs/ot-2/docs/installation/unboxing.md:
```diff
@@ -0,0 +1,74 @@
+  - Visually check the deck and windows; if there are any significantly scratched, broken, or bent components, contact Opentrons Support.
+  - Visually check the gantry; if it is uneven or slanted, contact Opentrons Support.
+  - Move the pipette head in the x and y direction; if it makes any loud/abnormal noises or is difficult to move, contact Opentrons Support.
+  - Clean and sanitize the robot. <font color="red">REMINDER: LINK TO CLEANING SECTION HERE</FONT>
```
note is for followup PR, i presume.

jwwojak replied:
Yes, that's just a note to myself and to future reviewers for when this is more complete. It stands out so we can't miss it.

ecormany commented on docs/ot-2/docs/installation/first-run.md:
```diff
@@ -0,0 +1,84 @@
+
+## Making a LAN connection
+
+Along with a direct connection to a computer, you can also connect the OT-2 to your local area network (LAN) using the supplied ethernet cable. For a LAN, just connect the ethernet cable to the robot and a wall jack. You can also plug the ethernet cable into a nearby network switch or hub. After the robot is connected and powered on, it will appear under the **Devices** tab in the Opentrons App.
```
"Ethernet" always
```suggestion
Along with a direct connection to a computer, you can also connect the OT-2 to your local area network (LAN) using the supplied Ethernet cable. For a LAN, just connect the Ethernet cable to the robot and a wall jack. You can also plug the Ethernet cable into a nearby network switch or hub. After the robot is connected and powered on, it will appear under the **Devices** tab in the Opentrons App.
```

jwwojak replied:
Yikes, fixing now.

ecormany commented on docs/ot-2/docs/installation/first-run.md:
```diff
@@ -0,0 +1,84 @@
+    - EAP-TLS
+
+!!!note "Captive portals not supported"
+    The OT-2 cannot be used on or connect to a [captive portal network](https://en.wikipedia.org/wiki/Captive_portal). Typically, these are the kind of networks deployed at airports, hotels, and other public access points. If no other networks are available, use a direct USB or ethernet connection to manage your robot.
```
```suggestion
    The OT-2 cannot be used on or connect to a [captive portal network](https://en.wikipedia.org/wiki/Captive_portal). Typically, these are the kind of networks deployed at airports, hotels, and other public access points. If no other networks are available, use a direct USB or Ethernet connection to manage your robot.
```

ecormany commented on docs/ot-2/docs/installation/first-run.md:
```diff
@@ -0,0 +1,84 @@
+
+## Naming your robot
+
+Naming your robot lets you easily identify it in your lab environment. If you have multiple Opentrons robots on your network, make sure to give them unique names. Once you've confirmed your robot's name, the next step is to install and calibrate a pipette.
```
Unlike on Flex, you aren't automatically prompted to do this. So if we keep this section, we should give instructions. 
1. Go to your robot > Robot settings > Advanced
2. Click Rename robot

We recommend USB connections for renaming, or at least we did. Not sure how flaky it is over Wi-Fi on an OT-2 these days.

jwwojak replied:
Good observation. I will add these instructions.

ecormany commented on docs/ot-2/docs/installation/instruments.md:
```diff
@@ -0,0 +1,22 @@
+title: "Opentrons OT-2: Instrument Installation and Calibration"
+---
+
+After the initial robot setup, your next step is to attach and calibrate a pipette. The OT-2 has two pipette mounts, each can hold a GEN2 1-channel or 8-channel pipette. The information in this section will help you get started; the Opentrons App provides detailed, illustrated instructions about the attachment and calibration workflows. To begin:
```
I think it's right to pretend that GEN2 are the only extant pipettes…but part of pretending so is not needing to distinguish between the generations. So I would remove "GEN2" here.

Also, "1-channel" and "8-channel" are absolutely accurate and descriptive, but the OT-2 parlance is "single-channel" and "multi-channel". 

Also also, I see your semicolon 🤓

jwwojak replied:
Will revised as recommended. I'm slowly coming around to the semi-colon. _don't fear the reaper_ plays distantly in the background.

ecormany commented on docs/ot-2/docs/installation/instruments.md:
```diff
@@ -0,0 +1,22 @@
+3. In the Pipettes and Modules section for your device:
+
+    - Select the left or right pipette mount.
+    - Click the 3-dot menu (⋮), and then click **Attach pipette**. The gantry will move towards the front of the deck and the mount will lower.
```
"three-dot" [per the style guide](https://opentrons.atlassian.net/wiki/spaces/RPDO/pages/980746400/Opentrons+UX+Style+Guide#Three-dots-(vertical-ellipsis)).

jwwojak replied:
Fixing that. Change to:

> three-dot menu

ecormany commented on docs/ot-2/docs/installation/relocation.md:
```diff
@@ -0,0 +1,34 @@
+A long-distance move transports your OT-2 off the grounds of your university, facility, or institution. In this case, you must lock the gantry and carefully pack the OT-2 to protect it from environmental elements, shocks, and rough movements that may occur in transit. For long distance moves, Opentrons recommends that you:
+
+- Disconnect the power cable from the robot.
+- Disconnect the ethernet cable from the robot and your computer.
```
📧 

```suggestion
- Disconnect the Ethernet cable from the robot and your computer.
```

ecormany commented on docs/ot-2/docs/installation/relocation.md:
```diff
@@ -0,0 +1,34 @@
+
+Your OT-2 is a precision scientific instrument designed to exacting tolerances. As a result, you should treat it with care when relocating it within your local work area or sending it across the country. This means following the guidance provided here and using your own common sense about how to transport an expensive piece of laboratory equipment. Bottom line: when moving your OT-2, err on the s ...
+
+If you have questions or concerns about relocating your robot, contact us at support@opentrons.com.
```
could link email address

jwwojak replied:
Adding link.

ecormany reviewed (approved):
Handful of very small things. Then go ahead and merge this into the higher-level branch!

jwwojak committed:
Apply suggestions from code review

Committing reviewer suggestions. Will go back in and make corrections based on other comments next.

Co-authored-by: Ed Cormany <edward.cormany@opentrons.com>

jwwojak reviewed (commented):
Thanks for the review. Will make changes.

jwwojak committed:
Making reviewer changes

jwwojak committed:
working on reviewer changes
